### DISCUSSION TITLE
AMS Filter Plugin Integration || React JS || Node JS || AMS

### DISCUSSION THREAD

**Discussion by aviJainGamermatic** [ant-media/Ant-Media-Server#5798] [Freelancing]
Need a Freelance Developer to integrate AMS Filter Plugin in our React JS web application.

**Comment by tremzo** [comment]
@aviJainGamermatic can you provide some more details of exactly what you need? 

The filter plugin adds filters on the ongoing streams by using a REST API. So are you looking for someone to add some kind of functionality in your ReactJS application that triggers the filter plugin APIs?
